### PR TITLE
Bug/update icons

### LINKED ISSUE #6: Some /symbols are getting clipped out of the viewbox
- DrawoutFuse
- LowVoltageDrawoutBreaker

These should be re-exported to fit within the viewbox (100x100)

### LINKED ISSUE #58: Generator using a white G
#### Describe the bug
The generator icon ( https://github.com/pxblue/icons/blob/dev/svg/generator.svg )'s "G" is white, instead of clipped out of the dark circle. 

#### What is the expected behavior?
Should be clipped

#### Screenshots or links to minimum reproduction example
from docit's iconography page
image

Should be
image

### LINKED ISSUE #59: power circle outlined icon's "P" is off-center
#### Describe the desired behavior
Should be more like this (from DSM):
image

#### Describe the current behavior
image

#### Additional Context
Icons like this can probably be programmed as progress-icons

### LINKED ISSUE #60: Road variant should be renamed to road alt
#### Describe the desired behavior
https://github.com/pxblue/icons/blob/dev/svg/road_variant.svg should be renamed to road_alt to follow the convention of naming things to "_alt"

#### Is this request related to a current issue?
No

#### Additional Context
Notify the DSM maintainer to rename it on DSM too.

### PR DESCRIPTION
**Pull request by huayunh**
Fixes #58, fixes #59, fixes #60, fixes #6 

Changes proposed in this Pull Request:
- I renamed road_variant to road_alt. This might result in a major update?

### CODE REVIEW

**Comment by huayunh** [comment]
I need to talk to Eric to figure out what he did to export those symbols...

**Comment by joebochill** [comment]
YES! Excited to see this making it into R16. 

I think Eric just does Export As -> from illustrator to make the .svg files

**Comment by huayunh** [comment]
Yeah I did the same thing but get some crooked results. I need to talk to him. Fortunately these are only for the symbols

**Comment by huayunh** [comment]
I ended up having to do save instead of export to prevent round-up errors from Adobe illustrator, and open the svg in text editors to "minify" it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
+## v1.0.26
+
+-   Bug fixes
```
Comment (joebochill) [inline CHANGELOG.md:3]: Can we mention the specific fixes?
```suggestion
-   Fixed a few alignment and color issues
-   Renamed road_variant to road_alt for consistency
```

**Comment by huayunh** [comment]
I see what's going on, let me try doing it again

**Comment by huayunh** [comment]
I deleted generator.ai because you don't really need .ai files to edit svg.

**Comment by huayunh** [comment]
> The two breaker symbols have changed from 48x48 to 150x150 - is that the same as the other existing ones?
Where did they change to 150? They should still stay at 48.

**Comment by joebochill** [comment]
> I deleted generator.ai because you don't really need .ai files to edit svg.

I would recommend we leave it in for now, just so we're consistent. But we may want to put a story on the backlog to overhaul our icon design / publish process... I think it's about time.

> Where did they change to 150? They should still stay at 48.

If you look at the Raw view in the file compare tool:
image

**Comment by huayunh** [comment]
> I would recommend we leave it in for now, just so we're consistent. But we may want to put a story on the backlog to overhaul our icon design / publish process... I think it's about time.

Um. I checked into the file structure, and not all the svgs have an ai file. I think it is safe to leave those out.

**Pull request merged**